code/datums/mind.dm

```diff
@@ -556,12 +556,12 @@ datum/mind
 				if("clear")
 					if(src in ticker.mode.revolutionaries)
 						ticker.mode.revolutionaries -= src
-						current << "\red <FONT size = 3><B>You have been brainwashed! You are no longer a revolutionary!</B></FONT>"
+						current << "<span class='danger'><FONT size = 3>You have been brainwashed! You are no longer a revolutionary!</FONT></span>"

```

Comment:
<font> is a non-conforming feature according to the HTML5 spec and shouldn't be used: http://www.w3.org/TR/html5/obsolete.html#non-conforming-features